CHANGELOG.md

```diff
@@ -0,0 +1,153 @@
+- Ensure that all kernels and memory allocations are prefixed with `ArborX::` [\#362](https://github.com/arborx/ArborX/issues/362) and [\#380](https://github.com/arborx/ArborX/pull/380)
+- Improve performance of knn traversal [\#357](https://github.com/arborx/ArborX/pull/357)
+- Add new query overloads for the distributed tree [\#356](https://github.com/arborx/ArborX/pull/356)
+- Increase performance of the BVH construction [\#350](https://github.com/arborx/ArborX/pull/350)
+- Allow non device type template parameter for output views in query\(\) on distributed trees [\#349](https://github.com/arborx/ArborX/pull/349)
+
+**Fixed bugs:**
+
+- Fix double free when making copies of a distributed tree [\#369](https://github.com/arborx/ArborX/pull/369)
+- Resolve duplicate `Details::toBufferStatus(int)` symbol error downstream [\#360](https://github.com/arborx/ArborX/pull/360)
+- Fix narrowing conversion warnings [\#343](https://github.com/arborx/ArborX/pull/343)
+
+**Deprecations:**
+
+- Deprecate `DistributedSearchTree` in favor of `DistributedTree` [\#396](https://github.com/arborx/ArborX/pull/396)
+
+## [0.9-beta](https://github.com/arborx/arborx/tree/v0.9-beta) (2020-06-10)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v0.8-beta2...v0.9-beta)
+
+**New features:**
+
+- Enable user-defined callback in BVH search [\#166](https://github.com/arborx/ArborX/pull/166)
+- Make predicates sorting optional [\#243](https://github.com/arborx/ArborX/pull/243)
+- Use user-provided execution space instances [\#250](https://github.com/arborx/ArborX/pull/250)
+- Implement algorithm for finding strongly connected components (halo finder) [\#237](https://github.com/arborx/ArborX/pull/237)
+- Store bounding boxes in single-precision floating-point format [\#235](https://github.com/arborx/ArborX/pull/235)
+- Allow usage of CUDA-aware MPI [\#162](https://github.com/arborx/ArborX/pull/162)
+
+**Build changes:**
+
+- Require Kokkos 3.1 [\#268](https://github.com/arborx/ArborX/pull/268)
+- Require C++14 standard when building ArborX [\#226](https://github.com/arborx/ArborX/pull/226)
+- Require Kokkos CMake build [\#93](https://github.com/arborx/ArborX/pull/93)
+
+**Enhancements:**
+
+- Template BVH on memory space [\#251](https://github.com/arborx/ArborX/pull/251)
+- Add example for callbacks and lift requirement for tagging inline [\#325](https://github.com/arborx/ArborX/pull/325)
+- Enable building against Trilinos' Kokkos installation [\#156](https://github.com/arborx/ArborX/pull/156)
+- Add access traits CUDA example [\#107](https://github.com/arborx/ArborX/pull/107)
+- Let `BVH::bounds()` be a `KOKKOS_FUNCTION` [\#326](https://github.com/arborx/ArborX/pull/326)
+- Improve performance of the radius search [\#306](https://github.com/arborx/ArborX/pull/306)
+- Improve performance of the kNN search [\#308](https://github.com/arborx/ArborX/pull/308)
+- Deprecate `Traits::Access` in favor of `AccessTraits` [\#300](https://github.com/arborx/ArborX/pull/300)
```

Comment:
In version 1.0-rc0, deprecated features are in their own sections. We should be consistent.

Reply:
Fixed in [`8b565ab` (#548)](https://github.com/arborx/ArborX/pull/548/commits/8b565abc43d398de3d26d7be96157cb992150bba).